### PR TITLE
styles: added wish list page

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Hello guys I have completed my wish list Page please review it Thanks a lot everyone 
Live UI-https://deploy-preview-3--piston-parts.netlify.app/pages/wishlist.html

### CODE REVIEW

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                    <button class="btn-icon-sml nav-bottom-icons">
+                        <i class="fas fa-sign-in-alt"></i></button>
+                </li>
+                <li class="nav-bottom__list-item">
+                    <button class="btn-icon-sml nav-bottom-icons">
+                        <i class="fas fa-heart"></i></button>
+                </li>
+
+
+            </ul>
```
Comment (samyukthasudhakar) [inline pages/wishlist.html:64]: Extra spacing between lines.

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        
+       
+    </nav>
+ 
+    <div class="margin-2"></div>
+    <!-- Product page Grid Layout -->
+   
+       
+   <h1 class="wishlist-heading text-center">Wishlist(4)</h1>     
+<main class="products-main">
```
Comment (samyukthasudhakar) [inline pages/wishlist.html:75]: Indent alignment missed.

File: pages/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        <div class="card-vertical-text sub-text margin-2">
+            <span class="text-black margin-l-1 font-weight-900">₹ 5</span> <span class="text-line-through margin-l-1">₹8</span>
+            <span class=" margin-l-1 text-black smooth-square-radius background-orange padding-4">5% off</span>
+         </div>
+        <div class="card-element__bottom no-border">
+            <button class="btn btn-pri margin-1 width-70">Add to Cart</button>
+            
+          
+          
+        </div>
```
Comment (samyukthasudhakar) [inline pages/wishlist.html:94]: Try using prettier to tackle all of the spacing/indent issues.

**Review by MayankKumar10 — APPROVED** [review]
card Test is on the left side apply some margin-left or padding everything else LGDM

**Review by samyukthasudhakar — APPROVED** [review]
LGTM, nice work !

**Comment by Dev-AkshatGupta** [comment]
> LGTM, nice work !

Thanks a lot for reviewing

File: pages/wishlist.html (hunk 3 above)
Comment (Dev-AkshatGupta) [reply pages/wishlist.html:94]: > If prettier is not helping na, try changing your formatter to the default vs-code one. It helped me @Dev-AkshatGupta

Ok I will do this way then If it doesn't works thanks :)

Comment (Dev-AkshatGupta) [reply pages/wishlist.html:94]: Thanks for reviewing Samyuktha I will indent it properly

**Comment by Dev-AkshatGupta** [comment]
> Certain observations
> 
> * add gap: 5px or 10px to the parent of the cards
> * add padding too, it will make your UI look good
> * the code looks good overall.

Yeah i have done that gap changes and also for the margin change I have to refactor my card which I will do after a while
Thanks for reviewing

**Pull request merged**